content/how-to/ignore-rules/local/index.md

```diff
@@ -1,26 +0,0 @@
----
-title: "Configuring local ignore rules"
```

Comment:
Is there a replacement for this?

Reply:
ignore.yaml is the replacement, CLI will read the rules if that file is present